base/applications/drwtsn32/main.cpp

```diff
@@ -233,14 +331,16 @@ int main(int argc, char* argv[])
     }
 
     PrintBugreport(output, data);
+    if (Settings_GetShouldWriteDump() && HasPath) WriteMinidump(OutputPath.c_str(), data);
 
     TerminateProcess(data.ProcessHandle, data.ExceptionInfo.ExceptionRecord.ExceptionCode);
 
+    std::wstring FilenameW(Filename);
     std::string Message = "The application '";
     Message += data.ProcessName;
     Message += "' has just crashed :(\n";
     Message += "Information about this crash is saved to:\n";
-    Message += Filename;
+    Message += std::string(FilenameW.begin(), FilenameW.end());
     Message += "\nThis file is stored on your desktop.";
```

Comment:
This part is no longer true

Reply:
And if you are adding a resource file, you might as well make this a message resource.